### DISCUSSION TITLE
Initial release

### DISCUSSION THREAD

**Discussion by mikulash** [mikulash/Octoprint-detector2#2] [Announcements]
first release, hope it will go well

<hr /><em>This discussion was created from the release <a href='https://github.com/mikulash/Octoprint-detector2/releases/tag/0.1.0'>Initial release</a>.</em>

**Comment by yeyuan98** [comment]
Genius idea, great work. Put the load on a PC rather than the pi. Works flawlessly with remote access octoeverywhere too.

Just need a spaghetti test case myself!
